index.html

```diff
@@ -148,8 +148,31 @@ <h4>Code meets data</h4>
         </div>
     </div>
 </section>
-<section id="current-use">
+<section id="voila">
     <div class="section-white">
+        <div class="voila-preview">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-6">
+                        <img class="img-responsive" src="assets/voilapreview.png" id="portal" alt="examples of Voilà dashboards"/>
+                    </div>
+                    <div class="col-md-6 nb-highlight-text">
+                        <h3>Voilà turns Jupyter notebooks into standalone web applications.</h3>
+                        <h4 class="nb-desc">
+                            Voilà can convert a Jupyter Notebook into an interactive dashboard that allows you to share your work with others. It is secure and customizable, giving you control over what your readers experience.
```

Comment:
"Voilà can convert a Jupyter Notebook into an interactive dashboard that allows you to share your work with others. It is secure and customizable, giving you control over what your readers experience."

Voilà helps you communicate insights, by transforming a Jupyter Notebook into a stand-alone web application you can share. It gives you control over what your readers experience in a secure and customizable interactive dashboard.

Reply:
Nice! Using your new sentence!